lib/util.js

```diff
@@ -517,7 +517,7 @@ function formatCollectionIterator(ctx, value, recurseTimes, visibleKeys, keys) {
   var nextRecurseTimes = recurseTimes === null ? null : recurseTimes - 1;
   var vals = mirror.preview();
   var output = [];
-  for (let o of vals) {
+  for (const o of vals) {
```

Comment:
I'd say it's definitely not safe to use `const` for loop vars and this looks like a rule and/or v8 bug to me. Spidermonkey also says no to this:

````
< for (const a of [1,2,3]) console.log(a)
> SyntaxError: invalid for/in left-hand side
````